src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/edit-video-unit/edit-video-unit.component.html

```diff
@@ -0,0 +1,9 @@
+<div *ngIf="isLoading" class="d-flex justify-content-center">
+    <div class="spinner-border" role="status">
+        <span class="sr-only">Loading...</span>
```

Comment:
use translations

Reply:
Done in b9b15e406ad96fca7d157f730c6c695fe79062a3